web_copy_confirm/static/src/js/web_copy_confirm.js

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2018 DynApps <http://www.dynapps.be>
+// @author Stefan Rijnhart <stefan@opener.amsterdam>
+// @author Robin Conjour <rconjour@demolium.com>
+// License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+odoo.define("web_copy_confirm.web_copy_confirm", function (require) {
+    "use strict";
+
+    const Core = require("web.core");
+    const FormController = require("web.FormController");
+    const Dialog = require("web.Dialog");
+    const _t = Core._t;
+
+    return FormController.include({
+        init: function () {
+            this._super.apply(this, arguments);
+        },
```

Comment:
Is the override of the init necessary?

Reply:
No, it was not necessary.

Reply:
Wait, it's not necessary but it allows other modules to override this method so it's good practice to return it.

Reply:
Then, it's back :)

Reply:
Apologies, my response was to another remark of Pieter's:
```
+    return FormController.include({

The return is also not necessary here I believe?
```

It is this return that I would like to see reintroduced. You can remove the override of the `init` method.

Reply:
Added the return, and removed the init.